### ISSUE TITLE
Broken links for accentuated notes

### ISSUE THREAD

**Issue by 0rphee** [datopian/flowershow#565]
Hi! just started using this project, very cool!

So, I have at least one note, for which references to it seem broken, here's an example.
```html
<a class="internal new transclusion" href="History of Ancient Philosophy - L">History of Ancient Philosophy - Lévystone Doc#1.3.1 Physical Appearance</a>
```
It also seems that for links with parentheses (ex. `my Note (parens).md`, this also happens (I have no examples, because I removed the parentheses to make the links work)

Thanks for your great work!

**Comment by rufuspollock** [comment]
Thanks for flagging and we'll take a look. @Gutts-n are you able to reproduce?

**Comment by Gutts-n** [comment]
@rufuspollock I'll take a look.

**Comment by Gutts-n** [comment]
In my tests here works well @0rphee, did you write the same name of the file that you want to link?
![image](https://github.com/datopian/flowershow/assets/57202549/c230dc88-b975-47e6-aac9-8aa5fb0b9130)
![image](https://github.com/datopian/flowershow/assets/57202549/3df0add8-d663-4acc-a4f4-8340cbad5f30)
![image](https://github.com/datopian/flowershow/assets/57202549/4d32d9c3-efb0-430f-9178-41091b619504)

**Comment by 0rphee** [comment]
@Gutts-n sorry for the delay. 

Yes, the note has the same name, the name of the note is the same as whats inside the <a> tag.

As I had showed in the initial example, the `href` value, seems to get cut off. The full name of the the file is 
<img width="583" alt="Screenshot 2023-10-09 at 3 00 25 p m" src="https://github.com/datopian/flowershow/assets/79347623/a459de56-99ce-4ec9-a79d-f3536a723d7c">

The full link in the href should be (I think) `History of Ancient Philosophy - Lévystone Doc#1.3.1 Physical Appearance`, but using my previous example again, however, this is the html that is produced.

> `<a class="internal new transclusion" href="History of Ancient Philosophy - L">History of Ancient Philosophy - Lévystone Doc#1.3.1 Physical Appearance</a>`

**Comment by Gutts-n** [comment]
Hey @0rphee we updated our lib that parses these URLs, update this package "@portaljs/remark-wiki-link" to "^1.1.1" in your project and let me know if worked.

**Comment by 0rphee** [comment]
I've updated it @Gutts-n .Still not working though.

With the same file this is the result of the link to the same file: `<a class="internal new transclusion" href="History of Ancient Philosophy ">History of Ancient Philosophy - Lévystone Doc#1.3.1 Physical Appearance</a>` 

Now instead of cutting of at the `é`, it cuts off at the `-`

**Comment by Gutts-n** [comment]
@0rphee Yep, I already fixed this new problem, but I need to someone do the code review for me. If the new change breaks your project feel free to back to the 1.1.0 version until we don't merge the new changes.

**Comment by Gutts-n** [comment]
@0rphee we merged the fix, you can use version ^1.1.2 of the package now.

**Comment by 0rphee** [comment]
@Gutts-n It works properly now. Thanks!
